web/viewer-geckoview.css

```
@@ -199,17 +205,13 @@ body {
 }
 
 .toolbarButton::before {
-  display: inline-block;
-  width: 100%;
-  height: 100%;
+  width: 24px;
+  height: 24px;
   content: "";
-  background-color: transparent;
-  backdrop-filter: invert(100%);
+  background-color: var(--toolbar-fg-color);
   mask-size: cover;
-}
-
-.toolbarButton::before {
-  opacity: var(--toolbar-icon-opacity);
+  margin-inline-start: 12px;
+  margin-inline-end: 8px;
```

Can't these be combined as follows?
```suggestion
  margin-inline: 12px 8px;
```